R/attr-nodes.R

```diff
@@ -0,0 +1,83 @@
+# CURLY ------------------
```

Comment:
I wasn't sure how to name this file.

Reply:
Considering how inconsistently I was at naming `asis-nodes` and other files, this fits in with the general aesthetic 😅